### DISCUSSION TITLE
Temperature probe readings too high until hardware is power cycled

### DISCUSSION THREAD

**Discussion by jslabotsky** [tagyoureit/nodejs-poolController#471] [General]
I guess this is more of a REM/RPi issue, but I thought I'd throw it here in case anyone has dealt with this. I am using the recommended Pentair temperature probe with an ADS1115 to monitor water temperature. It has been working great, but lately the temperature will abruptly jump 4-6 degrees F above actual, and rapidly fluctuate. Sometimes it goes back to the correct number after a period of time, but occasionally I have to power cycle the RPi and ADS1115 to get back to normal. Should I start replacing hardware or is there somewhere else I should look? Since REM is reading such small voltages, I wonder if I need to look for sources of induced current on the sensor line.

**Comment by rstrouse** [comment]
When it is doing that are the ADS readings jumping around or the Pi VCC voltage jumping around?  Make sure your wiring is making good contact.  I assume you installed a voltage divider on the 3v3 output from the pi.
